File: onebot-client/src/main/java/cn/evole/onebot/sdk/response/guild/GuildMetaByGuestResp.java
package cn.evole.onebot.sdk.response.guild;

import com.google.gson.annotations.SerializedName;
import lombok.Data;

/**
 * @author cnlimiter
 */
@Data
public class GuildMetaByGuestResp {

    @SerializedName("guild_id")
    private String guildId;

    @SerializedName("guild_name")
    private String guildName;

    @SerializedName("guild_profile")
    private String guildProfile;

    @SerializedName("create_time")
    private long createTime;

    @SerializedName("max_member_count")
    private long maxMemberCount;

    @SerializedName("max_robot_count")
    private long maxRobotCount;

    @SerializedName("max_admin_count")
    private long maxAdminCount;

    @SerializedName("member_count")
    private long memberCount;

    @SerializedName("owner_id")
    private String ownerId;

}

File: onebot/src/main/kotlin/sdk/response/guild/GuildMetaByGuestResp.kt
package cn.evolvefield.onebot.sdk.response.guild

import com.google.gson.annotations.SerializedName
import lombok.Data

@Data
class GuildMetaByGuestResp {
    @SerializedName("guild_id")
    var guildId = ""

    @SerializedName("guild_name")
    var guildName = ""

    @SerializedName("guild_profile")
    var guildProfile = ""

    @SerializedName("create_time")
    var createTime = 0L

    @SerializedName("max_member_count")
    var maxMemberCount = 0L

    @SerializedName("max_robot_count")
    var maxRobotCount = 0L

    @SerializedName("max_admin_count")
    var maxAdminCount = 0L

    @SerializedName("member_count")
    var memberCount = 0L

    @SerializedName("owner_id")
    var ownerId = ""
}
